packages/grafana-ui/src/components/Layout/Box/Box.mdx

```diff
@@ -32,7 +32,7 @@ If you need layout styles, use the Stack or Grid components instead.
         [cssProp]: theme.spacing(val),
     })),
    ```
-4. Add it to the `Box` story in `Box.internal.story.tsx`, by explicity adding it to `Basic.argTypes`
+4. Add it to the `Box` story in `Box.internal.story.tsx`, by explicity adding it to `Basic.ArgTypes`
```

Comment:
we sure we wanna change the casing here? the casing in the story is `argTypes`

Reply:
Nope. Reverted this. Thanks!